db/migrate/20231123105540_add_fields_for_new_document_numbering.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class AddFieldsForNewDocumentNumbering < ActiveRecord::Migration[7.0]
+  def change
+    change_table :organizations, bulk: true do |t|
+      t.integer :document_numbering, default: 0, null: false
+      t.string :document_number_prefix, null: true
+    end
+    add_column :invoices, :organization_sequential_id, :integer, null: false, default: 0
+
+    reversible do |dir|
+      dir.up do
+        execute <<-SQL
+          UPDATE organizations
+          SET document_number_prefix = UPPER(CAST(SUBSTRING(name, 1, 3) AS VARCHAR)) || '-' || UPPER(CAST(RIGHT(id::TEXT, 4) AS VARCHAR))
+          WHERE document_number_prefix IS NULL;
+        SQL
+
+        execute <<-SQL
+					WITH ordered_organization_invoices AS (
```

Comment:
I have a strange indentation, can you check on your side?

Reply:
Hmm, in my IDE looks good 🤔